automation/DeployBrowsers/deploy_firefox.py

```diff
@@ -237,6 +202,6 @@ def deploy_firefox(status_queue, browser_params, manager_params, crash_recovery)
     else:
         raise RuntimeError("Unable to identify Firefox process ID.")
 
-    status_queue.put(("STATUS", "Browser Launched", (int(pid), profile_settings)))
+    status_queue.put(("STATUS", "Browser Launched", (int(pid))))
```

Comment:
```suggestion
    status_queue.put(("STATUS", "Browser Launched", int(pid)))
```
As you only have one element here and no comma no tuple gets created.